exchanges/orderbook/orderbook_test.go

```diff
@@ -716,17 +689,17 @@ func BenchmarkSortAsksAscending(b *testing.B) {
 	s := deploySliceOrdered()
 	for i := 0; i < b.N; i++ {
 		ts := append(s[:0:0], s...)
-		SortAsks(ts)
+		ts.SortAsks()
 	}
 }
 
 // 12565	     97257 ns/op	   49208 B/op	       3 allocs/op
```

Comment:
Unfortunately, all of these benchmarks are now an order of magnitude greater than they used to be (at least on my machine™) . Please update so people can get a more accurate indication of what they are for the any future changes to compare to.

Reply:
Will redo these all

Reply:
On my machine, they are about the same 🤷 CPU: Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz. Changed the ones that needed the change.

Reply:
Im actually getting faster on that actual benchmark: `15340	     75205 ns/op	   49198 B/op	       3 allocs/op`

Reply:
To be clearer, here's my comparison:
| Benchmark                   | Commented benchmark                                          | Resulting benchmark                                          | Difference in ns/op |
| --------------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------- |
| BenchmarkReverse            | `705985	      1856 ns/op	       0 B/op	       0 allocs/op` | `521190	      2185 ns/op	       0 B/op	       0 allocs/op` | +329                |
| BenchmarkSortAsksDecending  | `20209	     56385 ns/op	   49189 B/op	       2 allocs/op` | `20868	     57500 ns/op	   49180 B/op	       2 allocs/op` | +1115               |
| BenchmarkSortBidsAscending  | `14924	     79199 ns/op	   49206 B/op	       3 allocs/op` | `14114	    104291 ns/op	   49199 B/op	       3 allocs/op` | +25092              |
| BenchmarkSortAsksStandard   | `9842        133761 ns/op      49194 B/op         2 allocs/op` | `9694	    120073 ns/op	   49186 B/op	       2 allocs/op` | -13588              |
| BenchmarkSortBidsStandard   | `7058        155057 ns/op      49214 B/op         3 allocs/op` | `8169	    160649 ns/op	   49204 B/op	       3 allocs/op` | +5592               |
| BenchmarkSortAsksAscending  | `20565        57001 ns/op      49188 B/op         2 allocs/op` | `18172	     65505 ns/op	   49181 B/op	       2 allocs/op` | +8504               |
| BenchmarkSortBidsDescending | `12565        97257 ns/op      49208 B/op        3 allocs/op` | `13639	     88840 ns/op	   49199 B/op	      3 allocs/op` | -8417               |
| BenchmarkDuplicatingSlice   | `923154        1169 ns/op       4096 B/op         1 allocs/op` | `83696	     12014 ns/op	   49153 B/op	      1 allocs/op` | +10845              |
| BenchmarkCopySlice          | `705922       1546 ns/op       4096 B/op         1 allocs/op` | `93628	     14365 ns/op	   49153 B/op	      1 allocs/op` | +12819              |

Reply:
There is always variance on the run, it's a benchmark and could swing wildly between go versions and hardware variability. I can delete the comments on the benchmarks and remove BenchmarkDuplicatingSlice with BenchmarkCopySlice as there isn't much difference anymore and we aren't exactly using them. The others can stay for reference i.e. why we would use reverse instead of sort functionality, but if you want all these deleted I can. 🤷

Reply:
Nah,you're right, its not really something worth focusing on. I came at it from the perspective of if someone were to make changes and then compare it to the comments, one could lead to think they've impacted things more than reality